src/Command/AnalyseCommand.php

```diff
@@ -272,6 +272,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 			}
 			if ($analysisResult->hasInternalErrors()) {
 				$inceptionResult->getStdOutput()->getStyle()->error('An internal error occurred. Baseline could not be generated. Re-run PHPStan without --generate-baseline to see what\'s going on.');
+				if ($output->isVerbose()) {
+					$inceptionResult->getStdOutput()->getStyle()->error(sprintf("Internal error list:\n-%s", implode("\n-", $analysisResult->getInternalErrors())));
```

Comment:
This isn't ideal. Ideally you'd use the error formatter here. You can see how it's invoked a few lines below: https://github.com/phpstan/phpstan-src/blob/69a300770c8f243a2ebea85102eb0976bf976f56/src/Command/AnalyseCommand.php#L360-L376

Reply:
@ondrejmirtes

Do you mean it's better to use table formatter for a list of errors? Or use AnalysisResult? Or something different?

Reply:
Yes, you need to creste AnalysisResult with just the internal errors and pass it to TableErrorFormatter.